roles/openshift_storage_cinder/tasks/main.yml

```diff
@@ -0,0 +1,17 @@
+---
+- name: Setup RDO repositories
+  yum: name=https://rdoproject.org/repos/rdo-release.rpm state=present
+
+- name: Install Nova and Cinder clients
+  yum: name=python-novaclient,python-cinderclient state=present
+
+- name: Create the cinder volume creation script
+  template:
+    dest: /usr/bin/create-openshift-cinder-volume
+    src: create-openshift-cinder-volume.j2
+    mode: 0755
```

Comment:
Instead of using a script here, this seems like an Ansible module would be a better fit here. Modules can be embedded in roles by adding a `library` directory to the role.

Docs on creating an Ansible module: http://docs.ansible.com/ansible/developing_modules.html